src/qibo/tests/test_abstract_circuit.py

```diff
@@ -117,6 +117,19 @@ def gen():
     assert isinstance(c.queue[5], gates.CNOT)
     assert isinstance(c.queue[7], gates.H)
 
+def test_circuit_add_repeated_execution():
```

Comment:
A new line is missing to have two line spacing between tests.
```suggestion

def test_circuit_add_repeated_execution():
```